### PR TITLE
chore: add tags and description for viz gallery 

### PR DESCRIPTION
**Pull request by stephenLYZ**
🏆 Enhancements
related https://github.com/apache/superset/issues/15720

### CODE REVIEW

File: packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Arc/index.js [hunk 1]
```diff
@@ -22,11 +22,14 @@ import transformProps from '../../transformProps';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Map'),
   credits: ['https://uber.github.io/deck.gl'],
-  description: '',
+  description:
+    'Plot the distance (like flight paths) between origin and destination.',
```
Comment (suddjian) [inline packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Arc/index.js:28]:
```suggestion
    t('Plot the distance (like flight paths) between origin and destination.'),
```

File: packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Hex/index.js [hunk 2]
```diff
@@ -22,11 +22,14 @@ import transformProps from '../../transformProps';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Map'),
   credits: ['https://uber.github.io/deck.gl'],
-  description: '',
+  description:
+    'Overlays a hexagonal grid on a map, and aggregates data within the boundary of each cell.',
```
Comment (suddjian) [inline packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Hex/index.js:28]:
```suggestion
    t('Overlays a hexagonal grid on a map, and aggregates data within the boundary of each cell.'),
```

File: packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Scatter/index.js [hunk 3]
```diff
@@ -22,12 +22,22 @@ import transformProps from '../../transformProps';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Map'),
   credits: ['https://uber.github.io/deck.gl'],
-  description: '',
+  description:
+    'A map that takes rendering circles with a variable radius at latitude/longitude coordinates',
```
Comment (suddjian) [inline packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Scatter/index.js:28]:
```suggestion
    t('A map that takes rendering circles with a variable radius at latitude/longitude coordinates'),
```

File: packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Screengrid/index.js [hunk 4]
```diff
@@ -22,12 +22,20 @@ import transformProps from '../../transformProps';
 import controlPanel from './controlPanel';
 
 const metadata = new ChartMetadata({
+  category: t('Map'),
   credits: ['https://uber.github.io/deck.gl'],
-  description: '',
+  description:
+    'Aggregates data within the boundary of grid cells and maps the aggregated values to a dynamic color scale',
```
Comment (suddjian) [inline packages/superset-ui-legacy-preset-chart-deckgl/src/layers/Screengrid/index.js:28]:
```suggestion
    t('Aggregates data within the boundary of grid cells and maps the aggregated values to a dynamic color scale'),
```

**Comment by junlincc** [comment]
Thanks for adding tags to the Deck.gl charts 🙏

**Comment by suddjian** [comment]
I’m just gonna merge this. Looks like Netlify is borked.

**Pull request merged**